docs/workflows.md

```diff
@@ -46,66 +46,51 @@ Enter a version number in the format `v1.2.3`, then hit the green `Run workflow`
-  # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
-  USE_GIT_LFS: false
-```
-
+  <summary>Environment variables section of the release workflow</summary>
+
+  ```yaml
+  env:
+    # The base filename of the binary produced by `cargo build`.
+    BINARY: bevy_template
+    # The name to use for the packaged application produced by this workflow.
+    PACKAGE_NAME: bevy-template
+    # The itch.io page to upload to, in the format: `user-name/project-name`.
+    # Comment this out to disable.
+    ITCH_TARGET: the-bevy-flock/bevy-template
+    # The organization or author that owns the rights to the game.
+    OWNER: the-bevy-flock
+    # The path to the assets directory.
+    ASSETS_DIR: assets
+    # Whether packages produced by this workflow should be uploaded to the Github release.
+    UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
+    # Before enabling LFS, please take a look at GitHub's documentation for costs and quota limits:
+    # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
+    USE_GIT_LFS: false
+  ```
 </details>
 
-In any case, make note of the value used for `ITCH_TARGET`, as you will need it later.
-
-### Set up permissions
-
-In your GitHub repository, navigate to `Settings > Actions > General`:
-
-![A screenshot showing how to navigate to the general GitHub Actions settings](./img/workflow-settings.png)
-
-Set `Workflow permissions` to `Read and write permissions`, then hit `Save`:
-
-![A screenshot showing where to change workflow permissions](./img/workflow-settings-permissions.png)
-
 ### Set up itch.io upload
 
-#### Create page
-
-Create your itch.io page with the same name as is used in the `ITCH_TARGET` variable in [release.yaml](../.github/workflows/release.yaml).
-By default, this is the same as the project name.
-Also set `Kind of project` to `HTML`.
-
-On your project page, you'll see a warning saying "There was a problem loading your project: No file provided to embed". This is because we haven't uploaded a release yet. Don't worry about it, we will fix that in a moment.
-
 #### Add butler credentials
 
-In your GitHub repository, navigate to `Settings > Secrets and variables > Actions`:
+<details>
+  <summary>In your GitHub repository, navigate to `Settings > Secrets and variables > Actions`</summary>
 
-![A screenshot showing where to add secrets in the GitHub Actions settings](./img/workflow-secrets.png)
+  ![A screenshot showing where to add secrets in the GitHub Actions settings](./img/workflow-secrets.png)
+</details>
 
-Hit `New repository secret` and enter the following, then hit `Add secret`:
+Hit `New repository secret` and enter the following values, then hit `Add secret`:
 
 - **Name:** `BUTLER_CREDENTIALS`
 - **Secret:** Your [itch.io API key](https://itch.io/user/settings/api-keys) (create a new one if necessary)
 
-#### Select the web build
+#### Set up itch.io project
```

Comment:
This was above so that it was closer to the section that references the `ITCH_TARGET`

Reply:
Hmm, I see how it's simpler this way. Let's leave it like this.

Reply:
We link to it so no need. Putting this below means that the GitHub repo is fully set up by the time you start setting up your itch.io prjoect, which allows the two sections to be combined into one (Create page + Select the web build), avoiding the awkward "there is an error but we'll fix it later".